### PR TITLE
portfolio desktop version

### PR DESCRIPTION
**Pull request by bigmosi**
Create portfolio desktop version using css media query.
Create css animation and transition.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,180 +19,207 @@
...
+          <li>
+            <a href="#"> <img src="assets/desktop-image/twitter.svg" alt="" class="image12" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="assets/desktop-image/m.svg" alt="" class="image12" /></a>
+          </li>
+        </ul>
+        <div class="my-work">
+          <h3 class="work">MY RECENT WORK</h3>
+        </div>
```
Comment (elmar8287) [inline index.html:113]: - [ ] Please, consider aligning the header "RECENT WORK" to the left, as it is in the Figma template.

> Your design
Screen Shot 2022-02-26 at 10 09 15

> Figma design
Screen Shot 2022-02-26 at 10 09 33

File: index.html [hunk 2]
```diff
@@ -19,180 +19,207 @@
...
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="lang">
+          <p class="last-text1">css</p>
+          <p class="last-text1">html</p>
+          <p class="last-text1">bootstrap</p>
+          <p class="last-text1-1">ruby</p>
+        </div>
+        <button class="button-1" type="submit">See Project</button>
```
Comment (elmar8287) [inline index.html:187]: - [ ] Please, consider **all buttons** design with design given in Figma (for hover, pressed,...)
Screen Shot 2022-02-26 at 10 13 08

- In your design the color is different. This is the color setting, according to the Figma template:
```
Enabled: #7D73FC
Hover: #7D73FC
Pressed: #5E42D7
Disabled: #2A2B33
```

File: index.html [hunk 3]
```diff
@@ -19,180 +19,207 @@
...
+        </div>
+        <p class="last-text">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="lang">
+          <p class="last-text1">css</p>
+          <p class="last-text1">html</p>
+          <p class="last-text1">bootstrap</p>
+          <p class="last-text1-1">ruby</p>
```
[the remark this answers is not in the document]
Comment (BenMukebo) [reply index.html:136]: > * [ ]  Please, consider fixing the alignment of each project to left, as it is required in the Figma template.
> 
> > You design
> 
> Screen Shot 2022-02-26 at 09 53 36
> 
> > Figma design
> 
> Screen Shot 2022-02-26 at 09 57 29

- [ OPTIONAL] You have aligned some changes addressed, but as you can see on this image below, the button is overlapping its container, I strongly suggest you fix them up, consider applying this change to all of your `see project` buttons
 
btnsee

File: index.html [hunk 4]
```diff
@@ -19,180 +19,207 @@
...
-        <h3>Software Developer</h3>
-        <p class="multi13">
-          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have
-          a project you need coded, don’t hestiate to contact me.
-        </p>
+    <div class="sub-container">
+      <div class="behind-image">
+        <div class="image-text">
+          <div class="main-title"><h1 class="title5">Laura Jong</h1></div>
+          <div class="main-content">
```
[the remark this answers is not in the document]
Comment (BenMukebo) [reply index.html:45]: > * [ ]  Please, consider giving to main header the correct styling, as it is mentioned in the Figma template. First of all, the font should be fixed to `Roboto`. Also, check other options as described in the screenshot below:
> 
> Screen Shot 2022-02-26 at 10 07 29
> 
> > Your design
> 
> Screen Shot 2022-02-26 at 10 08 15

- [ ] Great work, fixed 👍🏻

File: index.html (hunk 1 above)
Comment (BenMukebo) [reply index.html:113]: - [ ] Fixed, 👍🏻

File: index.html [hunk 5]
```diff
@@ -202,33 +229,34 @@ <h3 class="contact-info">CONTACT US</h3>
       <input type="email" name="email" id="email" placeholder="Enter an email" required class="text-input" />
 
       <label for="textarea"></label>
-      <textarea cols="30" rows="3" id="textarea" placeholder="Enter text here" class="text-area" required maxlength="500"></textarea>
+      <textarea cols="30" rows="3" id="textarea" placeholder="Enter text here"  required maxlength="500"></textarea>
 
       <label id="submit"></label>
-      <input type="submit" value="Send Message" class="button-2">
+      <input type="submit" value="Send Message" class="button-1">
```
Comment (BenMukebo) [inline index.html:235]: - [ OPTIONAL] Consider adding a `margin-top` to this button for moving it a bit down from the form, I strongly recommend you make all of these changes to make your work more professional and clean

paddi

**Review by BenMukebo — APPROVED** [review]
### Hi *** @bigmosi, *** 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**